Colaborators

arodrigueze0215 opened the pull request:
fixed #85  Add colaborators section

arendondiosa committed:
Actualizado readme con estructura de muestra y Licencia del proyecto

xergioalex committed:
Base webpack configuration for new website

xergioalex committed:
README udpated

xergioalex committed:
PereiraJs logo added to README

xergioalex committed:
Dist version updated

xergioalex committed:
README updated

jonalvarezz committed:
Initial

jonalvarezz committed:
Adds Anchor Component

jonalvarezz committed:
Hero: text

jonalvarezz committed:
Anchor style fixes

jonalvarezz committed:
Content: support ligh mode

jonalvarezz committed:
Button: Reduce size and support secondary style

jonalvarezz committed:
Adds About Us section

jonalvarezz committed:
Create README.md

jonalvarezz committed:
👋

jonalvarezz committed:
Introduce new design scaffolding

jonalvarezz committed:
Update package-lock.json

pin3da committed:
Merge pull request #92 from pereirajs/2019

🔥Nuevo Diseño 2019

arodrigueze0215 committed:
add api

arodrigueze0215 committed:
change index page

arodrigueze0215 committed:
Load events from PTT api

arodrigueze0215 committed:
Add some transition in the hover link in the events
Add link for each event to PTT page

jdmorales committed:
Merge pull request #93 from pereirajs/Events

Added last events in page

arendondiosa committed:
Fixed font

jdmorales committed:
Merge pull request #94 from pereirajs/font

Fixed font

arodrigueze0215 committed:
Add colaborators name

Is missing the style, and the rest of the colaborator info.

arodrigueze0215 committed:
add style to item and awesome icon

arodrigueze0215 committed:
add style icons social network

arodrigueze0215 committed:
add images profile static

arodrigueze0215 committed:
Add more colaborators pictures

arodrigueze0215 committed:
add avatar colaborators and links

arodrigueze0215 committed:
enable links

add click on links

pin3da commented on package.json:
```diff
@@ -1,31 +1,23 @@
-  "version": "0.2.0",
-  "description": "página web de pereirajs.org",
+  "name": "pereirajs",
+  "version": "0.1.0",
```
La version deberia ser mayor que la anterior, solo por consistencia.

arodrigueze0215 replied:
Ok! Fixed

pin3da committed:
Fix mintor style changes and typo:

- Using space only (found some tabs).
- Rename Colaborators to Collaborators.
- Using single quotes for strings.

pin3da commented on components/collaborators/Item.jsx:
```diff
@@ -0,0 +1,77 @@
+          <div className='contentItem'>
+            <A> {name} </A>
+            <div className='contentIcons'>
+              {github && (<div className='awesomeIcons' onClick={this.clickLink.bind(this, github)}><FontAwesomeIcon icon={faGithub} /></div>)}
```
@arodrigueze0215 Si entiendo bien, aca te falta el prefijo de cada pagina. Por ejemplo vas a tener un link a "XergioAleX" en vez de "https://twitter.com/XergioAleX"

arodrigueze0215 replied:
Cada colaborador tiene una serie de enlaces que provienen desde la data.json allí está toda la ruta completa de cada perfil (Twitter, Github, Linkedin, portafolio) y en esta clase Item.jsx llegan como parametros en los `props` para que simplemente yo haga un redirect de esas urls.

pin3da commented on components/collaborators/Item.jsx:
```diff
@@ -0,0 +1,77 @@
+  }
+
+  clickLink (link) {
+    console.log('link', link);
```
Puedes borrar este debug porfa?

arodrigueze0215 replied:
Sorry me..!!

pin3da commented on components/collaborators/Item.jsx:
```diff
@@ -0,0 +1,77 @@
+    );
+  }
+
+  clickLink (link) {
```
`redirectWindow.location` no parece generar ninguna accion. Esta bien esta parte?

arodrigueze0215 replied:
Que raro, en que Browser lo probaste? A mi me funciona en Firefox y en Brave bien

arodrigueze0215 replied:
Ya vi el error, me hizo falta un commit... Sorry

pin3da commented on components/hero/Navbar.js:
```diff
@@ -0,0 +1,36 @@
+          Eventos
+        </a>
+      </li>
+      <li className='ph2'>
```
Podrias agregar los collaboradores aca tambien?

arodrigueze0215 replied:
Tienes razón hizo falta el link en el Nav... I will

pin3da commented on components/collaborators/Item.jsx:
```diff
@@ -0,0 +1,77 @@
+                    }
+                    .itemCollaborator {
+                        display:flex;
+                        background: url('/static/collaborators/alejandro_rendon_.JPG');
```
Deberia estar basado en los datos del collaborator.

arodrigueze0215 replied:
Pffff esto fue un olvido mio, el background lo estoy pasando inline y olvidé quitar esa linea... Sorry

arodrigueze0215 committed:
fixed #85

arodrigueze0215 committed:
Revert "fixed #85"

This reverts commit 74f593d89e4fc4828118d4b9c35451d7c71e6332.

arodrigueze0215 committed:
Merge branch 'Colaborators' of https://github.com/pereirajs/pagina into Colaborators

arodrigueze0215 committed:
Fixed #85

cronopio commented on CNAME:
```diff
@@ -1 +0,0 @@
-pereirajs.org
```
Necesitamos borrar esto? Cloudflare esta sirviendo ya todo? porque creeria que si aun dependemos de Github Pages se podria dañar si esto se elimina. Igual estoy desactualizado de cosas, me cuentan

cronopio commented:
Este PR se debe cerrar y abrir uno nuevo ya que se abrio para hacer merge en `gh-pages` y debe hacerse merge a `develop`
